### PR TITLE
Codeine 245

### PR DESCRIPTION
**Pull request by ezraroi**
This change is [Reviewable](https://reviewable.io/reviews/codeine-cd/codeine/247)

### CODE REVIEW

File: src/peer/codeine/collectors/OneCollectorRunner.java [hunk 1]
```diff
@@ -93,6 +92,13 @@ private void runOnceCheckMinInterval() {
             log.info("collectors are disabled");
             return;
         }
+        List<String> intersect = collectorInfo.collector_tags().stream().filter(node.tags()::contains)
+            .collect(Collectors.toList());
+        if (intersect.size() != collectorInfo.collector_tags().size()) {
```
Comment (lchayoun) [inline src/peer/codeine/collectors/OneCollectorRunner.java]:
```suggestion
        if (node.tags().containsAll(collectorInfo.collector_tags())) {
```

File: src/peer/codeine/collectors/OneCollectorRunner.java [hunk 2]
```diff
@@ -93,6 +92,13 @@ private void runOnceCheckMinInterval() {
...
+            log.info("Skipping collector, as node is missing some tags: " + collectorInfo.collector_tags().stream()
```
Comment (lchayoun) [inline src/peer/codeine/collectors/OneCollectorRunner.java]: I think we can just log the collector tags

File: pom.xml [hunk 3]
```diff
@@ -183,22 +183,22 @@
 		<dependency>
 			<groupId>org.eclipse.jetty</groupId>
 			<artifactId>jetty-server</artifactId>
-			<version>9.4.8.v20171121</version>
+			<version>9.4.11.v20180605</version>
```
Comment (lchayoun) [inline pom.xml]: is it working with that version?

Comment (ezraroi) [reply pom.xml]: yes

**Pull request merged**